backend/src/utils/notebookUtils.ts

```diff
@@ -168,7 +168,7 @@ export const assembleNotebook = async (
 
     const selectedImage = getImageTag(image, imageTagName);
 
-    imageUrl = `${selectedImage.image?.dockerImageRepo}:${selectedImage.tag?.name}`;
+    imageUrl = `${selectedImage.tag?.from_name}`;
```

Comment:
Ah, also same issue here -- must have missed that during first viewing. This will generate `'undefined'` -- not sure what the intended fallback here is. Best to handle optional properties more gracefully.

Reply:
This should be the from.name of an imagestream tag, which is always present, example here https://github.com/opendatahub-io/odh-manifests/blob/master/notebook-images/base/jupyter-datascience-notebook-imagestream.yaml#L22. I will look into that at this program-code level here. Tag.from.name should aways be present. https://itnext.io/variations-on-imagestreams-in-openshift-4-f8ee5e8be633